server/channels/store/sqlstore/channel_store.go

```diff
@@ -684,12 +709,12 @@ func (s SqlChannelStore) saveChannelT(transaction *sqlxTxWrapper, channel *model
 	var insert string
 	if s.DriverName() == model.DatabaseDriverMysql {
 		insert = `INSERT IGNORE INTO Channels
-		(Id, CreateAt, UpdateAt, DeleteAt, TeamId, Type, DisplayName, Name, Header, Purpose, LastPostAt, TotalMsgCount, ExtraUpdateAt, CreatorId, SchemeId, GroupConstrained, Shared, TotalMsgCountRoot, LastRootPostAt)
+		(` + strings.Join(channelSliceColumns(""), ", ") + `)
```

Comment:
I think we should revert the changes to this method since the hardcoded list of fields below isn't guaranteed to match the values returned by `channelSliceColumns`

Reply:
This is matched now via `channelToSlice` method.